automation/run-tests.sh

```diff
@@ -70,6 +70,13 @@ function install_nmstate {
     '
 }
 
+function mark_eth1_eth2_nm_managed {
+    docker_exec '
+      nmcli device set eth1 managed yes && \
```

Comment:
It might be better to make Nmstate do this if a user tries to set a state for an interface that requires this instead of hiding this in the CI setup.

Reply:
Good point.
Let's do that in https://nmstate.atlassian.net/browse/NMSTATE-17